app/cafe.py

```diff
@@ -0,0 +1,26 @@
+import datetime
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        print(f"Visitor data received: {visitor}")
+        if "vaccine" not in visitor:
+            print("Vaccine key missing. Raising NotVaccinatedError.")
+            raise NotVaccinatedError("Visitor is not vaccinated.")
+        today_date = datetime.date.today()
+        expiration_date = visitor["vaccine"]["expiration_date"]
+        if expiration_date < today_date:
+            raise OutdatedVaccineError("Visitor's vaccine is outdated.")
+        if "wearing_a_mask" not in visitor or not visitor["wearing_a_mask"]:
```

Comment:
You can use `get` method to simplify your code
```suggestion
        if not visitor.get("wearing_a_mask"):
```